Plugins/GRPCSwiftPlugin/plugin.swift

```diff
@@ -205,6 +211,12 @@ struct GRPCSwiftPlugin {
 
       // Add the outputPath as an output file
       outputFiles.append(protobufOutputPath)
+
+        if invocation.reflection == true {
+            let reflectionFile = file.replacingOccurrences(of: "grpc.swift", with: "grpc.reflection")
```

Comment:
The code above deliberately doesn't use this pattern because it's not necessarily correct. The filename could be "grpc.swift.foo.bar.grpc.swift" and this would incorrectly rename it to "grpc.reflection.foo.bar.grpc.reflection".

Reply:
Changed the logic here so drop the last `swift` and add `reflection` similar to how `proto` is dropped and `swift is appended.